en/02_Developer_Guides/15_Customising_the_Admin_Interface/06_Javascript_Development.md

```diff
@@ -132,71 +60,14 @@ Main advantages over simpler jQuery plugins are:
 *  Constructor/Destructor hooks
 *  Focus management and mouse interaction
 
-See the [official developer guide](http://jqueryui.com/docs/Developer_Guide) and other
-[tutorials](http://bililite.com/blog/understanding-jquery-ui-widgets-a-tutorial/) to get started.
-
-Example: Highlighter
-
-```js
-(function($) {
-  $.widget("ui.myHighlight", {
-    getBlink: function () {
-      return this._getData('blink');
-    },
-    setBlink: function (blink) {
```

Comment:
Our docs don't need to explain what jQueryUI widgets are good for or how to build them. Leave that to the official docs.